src/views/RecIM/RecIM.module.scss

```
@@ -1 +1,14 @@
 @import '../../vars';
+
+.card {
+  background-color: #014983;
+  color: #ffffff;
```

Make sure to avoid hard-coded color codes in the long term. 
```suggestion
  background-color: $primary-blue;
  color: $neutral-white;
```